Makefile

```diff
@@ -1,15 +1,25 @@
 NAME = oscap-anaconda-addon
 
-VERSION = 3.0.0
+VERSION = 0.35.0
```

Comment:
I propose to use versions targetting Fedora in the format `0.<fedora version>.<patch version component>`